packages/popover2/src/contextMenu2.tsx

```diff
@@ -109,7 +109,7 @@ export interface ContextMenu2Props
     /**
      * A limited subset of props to forward along to the popover generated by this component.
      */
-    popoverProps?: IOverlayLifecycleProps & Pick<Popover2Props, "popoverClassName" | "transitionDuration">;
+    popoverProps?: IOverlayLifecycleProps & Pick<Popover2Props, "popoverClassName" | "transitionDuration" | "popoverRef">;
```

Comment:
this line is probably too long and is failing formatting checks. run `yarn format` from the root of the repo.